src/core/init/initialize_media_source.ts

```diff
@@ -112,6 +112,8 @@ export interface IInitializeArguments {
      * higher-quality ones to have a faster transition.
      */
     enableFastSwitching : boolean;
+    /** Stategy when switching of audio track (smooth vs flush vs reload). */
```

Comment:
No biggie but for next time,it's not necessary to repeat the possibilities in the comment as they are explicitly typed in the code.

Reply:
Sure, I edit it right now